### DISCUSSION TITLE
Calculation of atom-projected phonon DOS with MODE=SCPH.

### DISCUSSION THREAD

**Discussion by jjma0924** [ttadano/alamode#60] [Q&A] [answered]
Dear Dr. Tadano,
    I am trying to calculate the anharmonic phonon dispersion and phonon density of states (PHDOS).  In the calculation, the anharmonic IFCs are estimated through LASSO technique. I can get the phonon dispersion and total PHDOS at different temperatures.  However,  when I calculate the atom-projected PHDOS，the setting of PDOS = 1 didn't  seem to work.  In this case, how to calculate the atom-projected PHDOS?
    Any suggestions would be greatly appreciated.   

Best,
Jiang Jiang

Following is the input file I used for the PHDOS calculation:
&general
  PREFIX = Si_scph
  MODE = SCPH
  FCSXML = Si_AIMD.xml
  NKD = 1; KD = Si
  MASS = 28.0855
  TMIN = 0; TMAX = 1000; DT = 100
/

&scph
 KMESH_SCPH        = 4 4 4
 KMESH_INTERPOLATE = 4 4 4
 SELF_OFFDIAG = 0
 RESTART_SCPH = 0
 MIXALPHA = 0.1
 MAXITER = 500
 TOL_SCPH = 1.0e-10
/

&analysis
 DOS = 1
 #PDOS = 1
/ 

&cell
  10.3339
  0.0 0.5 0.5
  0.5 0.0 0.5
  0.5 0.5 0.0
/

&kpoint
  2  
  20 20 20
/

**Comment by ttadano** [comment, marked as answer]
Thank you for reminding me of this limitation. As you have noticed, `PDOS =1` does not work in the SCPH mode. I will fix it as soon as possible. 

For the moment, please use the `dfc2` script in the tools directory and generate a new XML file containing the effective second-order force constants. Then, please run `anphon` code with `MODE = phonons` along with the new XML file specified in the `FCSXML` tag.

**Reply by jjma0924** [reply]
Dear Dr. Tadano,
   
   Thank you very much for your quickly response,  I will try it.
   
Best,
Jiang Jiang

**Reply by QuaziBhoi** [reply]
Dear Dr. Tadano

Any updated solution regarding this issue?

Best,
Shafayat
